src/web/lib/components/AppHeader/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import ReactSVG from 'react-svg';
+import { surveyUrl } from '../../../../lib/constants';
+
+import './index.scss';
+
+export const AppHeader = ({ hasExtension, appVersion = null }) => {
+  let survey = `${surveyUrl}?ref=app`;
+  if (appVersion) {
+    survey += `${survey}&ver=${appVersion}`;
+  }
+  return (
+    <div className="app-header">
+      <div className="app-header__content">
+        <div className="app-header__icon" />
+        <header>
+          <h1>Themer</h1>
+          <h3>A <a href="https://testpilot.firefox.com" target="_blank"
+           rel="noopener noreferrer">Firefox Test Pilot</a> Experiment</h3>
+        </header>
+      </div>
+      {hasExtension &&
+        <a href={ survey }
+           title="survey link"
+           className="app-header__survey"
+           target="_blank"
+           rel="noopener noreferrer">
+            <span>Feedback</span>
+            <ReactSVG
+                    style={{ fill: '#fff' }}
+                    path='../../../../images/feedback.svg'
```

Comment:
Oh wait, I think this needs a tweak: Should refer to this image path with an import ([for example](https://github.com/mozilla/ThemesRFun/blob/master/src/web/lib/components/ThemeColorsEditor/index.js#L9)) and then use that as the path ([for example](https://github.com/mozilla/ThemesRFun/blob/master/src/web/lib/components/ThemeColorsEditor/index.js#L21)) - that way, Webpack picks it up and processes it with a hashed filename.